searx/plugins/autodetect_search_language.py

```diff
@@ -0,0 +1,14 @@
+from langdetect import detect
+
+name = 'Autodetect search language'
+description = 'Automatically detect the query search language and switch to it.'
+preference_section = 'general'
+default_on = False
+
+js_dependencies = tuple()
+css_dependencies = tuple()
```

Comment:
What is the reason for these two variables with empty tuples?

Reply:
They were in the example in the docs, I there is no reason to keep them, I assumed it was a standard to have them in all plugins

Reply:
You can remove them. If you want to you can clean up the branch by squashing some commits together.

Reply:
Okay, if there's anything else you want me to do tell me, I'm on my phone atm, but I will go back to this PR tomorrow

Reply:
Done, hopefully everything is good now